Neos.Flow/Classes/Mvc/View/ViewInterface.php

```diff
@@ -26,53 +25,45 @@ interface ViewInterface
     /**
      * Sets the current controller context
      *
+     * @deprecated if you absolutely need access to the current request please assign a variable.
```

Comment:
Will this even show in IDEs with the function commented out? bit nasty I guess 🤷

Reply:
documenting this via 
> @method setControllerContext(ControllerContext $controllerContext) @deprecated
on the view-interface will not have the desired effect of annotating it as deprecated but would still allow goto clicking.

In the end all views extend the AbstractView which still contains a definition for `setControllerContext` (but deprecated) so i guess its fine?